src/network/mod.rs

```diff
@@ -224,25 +231,24 @@ impl GameSniffer {
         }
 
         if let Some(kcp) = kcp {
-            let commands = kcp
-                .receive_segments(kcp_seg)
-                .into_iter()
-                .filter_map(|data| self.receive_command(data))
-                .collect();
+            let mut commands = vec![];
+            for command in kcp.receive_segments(kcp_seg)? {
+                commands.push(self.receive_command(command)?);
```

Comment:
This is a behavior change, the previous code used to filter out the invalid commands, the new version bails out.